all-requirements.txt

```diff
@@ -10,3 +10,6 @@ tornado==4.2
 
 # For simpl.middleware.cors
 WebOb==1.4.1
+
+# MongoDB
+pymongo==2.8.1
```

Comment:
Yeah. I pip installed and got 3.0.3 and that broke a ton of things. As usual, I'm trying to port existing knowledge w/o spending innovation credits :-)